script/Arcueid/Day 4/04_00_ARC04_1B.txt

```diff
@@ -276,12 +297,12 @@ She's lacking her usual bravado. She must be worried about my health, in her own
 [sha:1e6b2eedf850eb157c6f39717fc38771513bc728]{
 -- Page 50, Offset 5684.
 -- 「おはよう。体のほうは、まあ、それなりに調子いいよ」
-"Good morning. I'm feeling much better today."
+"Good morning. As for my body, well, it could certainly be worse."
```

Comment:
I feel 'could be worse' is too strong, especially since Shiki is trying to alleviate her worries about him. The hesitant 'well' is sufficient in conveying that he's putting up a front imo.

> As for my body, well, it's in pretty good shape.